### ISSUE TITLE
Accessibility errors found in your template

### ISSUE THREAD

**Issue by wapyce** [TaylanTatli/Halve#61]
The user @carlsonsantana validate your site template "https://taylantatli.github.io/Halve/" and found these accessibility errors:
* Page https://taylantatli.github.io/Halve/:
  * Issue *e9053623-f85e-4a61-8489-cf48c1a1c4d0*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.85:1. Recommendation: change text colour to #767676.
    * **Selector**: `#homepage > div:nth-child(2) > div > div > p:nth-child(4)`
    * **Context**: `<p>If you have a question, find a ...</p>`
  * Issue *ac8643f4-3647-452c-8bca-db1d4a431800*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.85:1. Recommendation: change text colour to #767676.
    * **Selector**: `#homepage > div:nth-child(2) > div > div > p:nth-child(3)`
    * **Context**: `<p>To learn how to install and use...</p>`
  * Issue *41c7cf2c-fb19-48d5-a9a5-a3aa6fd3ada1*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.85:1. Recommendation: change text colour to #767676.
    * **Selector**: `#homepage > div:nth-child(2) > div > div > p:nth-child(2)`
    * **Context**: `<p>This theme is Jekyll port of <a...</p>`
  * Issue *4d42778e-3334-47ba-b59f-2f3390f211b2*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.A.EmptyNoId
    * **Message**: Anchor element found with no link content and no name and/or ID attribute.
    * **Selector**: `#homepage > div:nth-child(2) > a:nth-child(3)`
    * **Context**: `<a title="project" class="projects-menu-icon"> <span></span> </a>`
  * Issue *737d79de-85a4-4941-8122-548499f4a900*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.A.NoContent
    * **Message**: Anchor element found with a valid href attribute, but no link content has been supplied.
    * **Selector**: `#homepage > div:nth-child(1) > div > div > ul > li:nth-child(4) > a`
    * **Context**: `<a class="social-btn" href="http://stackoverflow.com/users/123456/username from a " http:="" stackoverflow.com="" users="" 123456="" username"="" link"="" target="_blank" rel="noopener noreferrer"><i class="fa fa-fw fa-stack-ove...</a>`
  * Issue *3487a88b-3b05-4811-9318-64e4dbc1e086*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.A.NoContent
    * **Message**: Anchor element found with a valid href attribute, but no link content has been supplied.
    * **Selector**: `#homepage > div:nth-child(1) > div > div > ul > li:nth-child(3) > a`
    * **Context**: `<a class="social-btn" href="http://github.com/username" target="_blank" rel="noopener noreferrer"><i class="fa fa-fw fa-github"><...</a>`
  * Issue *5debcb80-53ae-4037-b994-607a3ef53026*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.A.NoContent
    * **Message**: Anchor element found with a valid href attribute, but no link content has been supplied.
    * **Selector**: `#homepage > div:nth-child(1) > div > div > ul > li:nth-child(2) > a`
    * **Context**: `<a class="social-btn" href="http://twitter.com/username" target="_blank" rel="noopener noreferrer"><i class="fa fa-fw fa-twitter">...</a>`
  * Issue *b6f1f74f-2891-4e31-9e99-31f41f7e393d*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.A.NoContent
    * **Message**: Anchor element found with a valid href attribute, but no link content has been supplied.
    * **Selector**: `#homepage > div:nth-child(1) > div > div > ul > li:nth-child(1) > a`
    * **Context**: `<a class="social-btn" href="https://www.behance.net/username" target="_blank" rel="noopener noreferrer"><i class="fa fa-fw fa-behance">...</a>`
  * Issue *bcefdfca-ee09-490a-9f58-95aa6110d759*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_1.1_1_1.H30.2
    * **Message**: Img element is the only content of the link, but is missing alt text. The alt text should describe the purpose of the link.
    * **Selector**: `#homepage > div:nth-child(1) > div > div > a`
    * **Context**: `<a href="https://taylantatli.github.io/Halve" class="logo"><img src="https://taylantatli.g...</a>`
  * Issue *c93a3b7b-2de3-4211-bf09-1cafbb79e0f3*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle3.Guideline3_1.3_1_1.H57.2
    * **Message**: The html element should have a lang or xml:lang attribute which describes the language of the document.
    * **Selector**: `html`
    * **Context**: `<html><head> <meta charset="utf-8"> <...</html>`

You can check these accessibility errors using [pa11y](https://github.com/pa11y/pa11y).
You can view the [full validation results in our website](https://wapyce.herokuapp.com/validation/validations/2fcb5561-6af9-4807-8a68-8a9c4b03983b/).
